deployments/common/image/base.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - http://ssb.stsci.edu/astroconda
   - defaults
 dependencies:
-  #- bqplot-image-gl==1.4.1
```

Comment:
@jaytmiller I'm not sure why these packages needed to be specified here, but they were necessary in the latest jwebbinar build.

Reply:
I think packages in base.yml are only going into the base Python environment.

Aside from env cloning,  packages to be installed in all Python environments are either defined in common-env or common-scripts.